### PR TITLE
Add functionality

### PR DESCRIPTION
**Pull request by elysehadid**
Work in progress.

Clickup card for context: https://app.clickup.com/t/866a7qkzw

### CODE REVIEW

**Comment by elysehadid** [comment]
Stubbed everything out but the thinking things through part of this took longer than I thought, lolz.

File: pages/index.tsx [hunk 1]
```diff
@@ -1,93 +1,39 @@
...
 
-const viewerDocument = graphql(/* GraphQL */ `
-  query Viewer {
-    viewer {
-      id
-      name
-      status
-    }
-  }
-`)
```
Comment (adriantoddross) [inline pages/index.tsx:25]: What do you think about removing the boilerplate code and clean this up b4 implementing the rest of this?

Comment (adriantoddross) [reply pages/index.tsx:25]: Oops, meant to put this comment on index.tsx, not this file.

File: public/pose-downwardDog.webp
Comment (elysehadid) [inline public/pose-downwardDog.webp:1]: Shouts out to webp!!!!

File: pages/index.tsx [hunk 2]
```diff
@@ -1,93 +1,35 @@
...
+export default function Home() {
+  const [currentPoseIndex, setPoseIndex] = useState(0);
 
-const viewerDocument = graphql(/* GraphQL */ `
-  query Viewer {
-    viewer {
-      id
-      name
-      status
```
Comment (elysehadid) [inline pages/index.tsx:22]: Just removing the boilerplate code from the original file; Don't pay attention to this.

File: pages/index.tsx [hunk 3]
```diff
@@ -1,93 +1,35 @@
...
-      initialApolloState: apolloClient.cache.extract(),
-    },
-  }
+      <div>
+        <h2>{title}</h2>
+        <p>{description}</p>
+        <button type="button" onClick={handleShowNextImage}>
+          Show next pose
+        </button>
+      </div>
```
Comment (elysehadid) [inline pages/index.tsx:24]: What do you think about converting this div into a nested section element?

**Pull request merged**